scripts/prune-extra-keep-files.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+for EXERCISE_DIRECTORY in $(find exercises -mindepth 2 -maxdepth 2 -type d -name "src"); do
```

Comment:
👍 using "modern" command substitution.  ([why is it preferred?](http://mywiki.wooledge.org/BashFAQ/082)).